mex/common/ldap/connector.py

```diff
@@ -88,6 +88,30 @@ def _paged_ldap_search(
         )
         return list(entries)
 
+    def get_functional_accounts(
+        self, mail: str = "*", sAMAccountName: str = "*", **filters: str
+    ) -> Generator[LDAPActor, None, None]:
+        """Get LDAP functional accounts that match provided filters.
+
+        Some projects/resources declare functional mailboxes as their contact.
+
+        Args:
+            sAMAccountName: Account name
+            mail: Email address of the functional account
```

Comment:
```suggestion
            mail: Email address of the functional account
            sAMAccountName: Account name
```

keeping same order as signature may avoid confusion

Reply:
Done :)